jobs/openshift/cluster/brew/openshift-cluster-brew-image-sync-1835.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- job:
+    name: openshift-cluster-brew-image-sync-1835
```

Comment:
NOTE: Will need to remove `-1835` from the job template name when this job is ready to be used.

Reply:
I think you can just remove it now and update the real job. I thought it was already updated tbh.